test/scripts/base/can_find_sdk.py

```diff
@@ -18,5 +18,5 @@
 # check sdk not empty
 files_in_sdk_directory = os.listdir(PATH_TO_SDK_IN_VM)
 print(files_in_sdk_directory)
-if len(files_in_sdk_directory) == 0:
+if not files_in_sdk_directory:  # (seq is false if empty)
```

Comment:
The code should not need comments. As this is not performance critical, I would keep the more verbose `len(x) == 0` version.

Reply:
PEP8 says you should prefer the second way and should not use the first.  
I agree code should not need comments though...

Reply:
Maybe if we rename `files_in_sdk_directory` to `list_of_files_in_sdk_directory` then we could change to use `not` instead of `len(x) == 0` and leave out the comment :)